src/file.hpp

```diff
@@ -113,7 +121,10 @@ namespace Sass {
 
   namespace File {
 
-    std::vector<Include> resolve_includes(const std::string& root, const std::string& file);
+    static std::vector<std::string> defaultExtensions = { ".scss", ".sass", ".css" };
```

Comment:
IMO it should move into C-API to be configurable. This is just a small step in that direction.